content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+=======
+Uruguay
+=======
+
+.. |DGI| replace:: :abbr:`DGI (Dirección General Impositiva)`
+.. |EDI| replace:: :abbr:`EDI (Electronic Data Interchange)`
+
+Introduction
+============
+
+With the Uruguayan localization, you can generate electronic documents with its XML, fiscal folio,
+electronic signature and connection to tax authority Dirección General Impositiva (DGI) through
+Uruware.
+
+The supported documents are:
+
+- Invoices, credit notes, debit notes;
+- e-ticket e-ticket credit notes, e-ticket debit notes;
+- Export e-invoice, export e-credit notes, export e-debit notes.
+
+The localization requires an Uruware account, which enables users to generate electronic documents
+within Odoo.
+
+Glossary
+------------
+
+The following terms are used throughout the Uruguayan localization:
+
+- **DGI**: *Dirección General Impositiva* is the government organization that enforces pay of
+  taxes in Uruguay.
+- **EDI**: *Electronic Data Interchange* refers to the sending of electronic documents.
+- **Uruware**: is the third-party organization that facilitates the interchange of electronic
+  documents between companies and the Uruguayan government.
+
+Configuration
+=============
+
+Modules installation
+--------------------
+
+:ref:`Install <general/install>` the following modules to get all the features of the Uruguayan
+localization:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Uruguay - Accounting`
+     - `l10n_uy`
+     - The default :doc:`fiscal localization package <../fiscal_localizations>`, adds accounting
+       characteristics for the Uruguayan localization, which represent the minimum configuration
+       required for a company to operate in Uruguay according to the guidelines set by the |DGI|.
+       The module's installation automatically loads: chart of accounts, taxes, documents types, and
+       tax support types.
+   * - :guilabel:`Uruguay Accounting EDI`
+     - `l10n_uy_edi`
+     - Includes all the technical and functional requirements to generate and validate
+       :doc:`Electronics Documents <../accounting/customer_invoices/electronic_invoicing>`, based on
+       the technical documentation published by the |DGI|. The authorized documents are: invoices,
+       credit notes and debit notes.
+
+.. note::
+   Odoo automatically installs the base module *Uruguay - Accounting* when a database is
+   installed with `Uruguay` selected as the country.
```

Comment:
We should mention that they would need to install the l10n_uy_edi afterwards (if they want to have electronic invoicing, of course).